ecosystem/indexer-grpc/indexer-grpc-manager/src/config.rs

```diff
@@ -20,6 +20,9 @@ pub(crate) struct ServiceConfig {
 pub struct IndexerGrpcManagerConfig {
     pub(crate) chain_id: u64,
     pub(crate) service_config: ServiceConfig,
+    pub(crate) self_advertised_address: String,
```

Comment:
nit: either give it a type other than string or use socketaddr/url.

Reply:
Added a type.